lib/capybara/node/finders.rb

```diff
@@ -53,23 +55,21 @@ def find(*args, **options, &optional_filter_block)
 
       ##
       #
-      # Find an {Capybara::Node::Element} based on the given arguments that is also an ancestor of the element called on. +ancestor+ will raise an error if the element
+      # Find an {Capybara::Node::Element} based on the given arguments that is also an ancestor of the element called on. {#ancestor} will raise an error if the element
       # is not found.
       #
-      # +ancestor+ takes the same options as +find+.
+      # {#ancestor} takes the same options as {#find}.
       #
       #     element.ancestor('#foo').find('.bar')
       #     element.ancestor(:xpath, './/div[contains(., "bar")]')
       #     element.ancestor('ul', text: 'Quox').click_link('Delete')
       #
-      # @param (see Capybara::Node::Finders#find)
-      #
       # @macro waiting_behavior
       #
-      # @option options [Symbol] match        The matching strategy to use.
-      #
-      # @return [Capybara::Node::Element]      The found element
-      # @raise  [Capybara::ElementNotFound]    If the element can't be found before time expires
+      # @overload ancestor([kind], locator, **options)
+      # @param (see #find)
```

Comment:
Does this actually link?? Also I don't really see the need for removing the return and raise from this methods declaration.